src/components/Header.jsx

```diff
@@ -1,23 +1,36 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { FaSmileWink } from 'react-icons/fa';
+import css from './Header.module.css';
 
 function Header() {
   return (
-    <div className="Header">
-      <h1>Bookstore</h1>
-      <ul>
-        <li>
-          <NavLink to="/" className="links">
-            Books
-          </NavLink>
-        </li>
-        <li>
-          {' '}
-          <NavLink to="Categories" className="links">
-            Categories
-          </NavLink>
-        </li>
-      </ul>
+    <div className={css.header}>
+
+      <div className={css.header_left_nav}>
+        <div>
+          <h1 className={css.logo}>
+            Bookstore
+          </h1>
+          <ul>
+            <li>
+              <NavLink to="/" className={css.links}>
+                Books
+              </NavLink>
+            </li>
+            <li>
+              {' '}
+              <NavLink to="Categories" className={`${css.links} ${css.disabled}`}>
+                Categories
+              </NavLink>
+            </li>
+          </ul>
+        </div>
```

Comment:
- [ ] Please ensure that the left and right margin on the `Header` component corresponds with the Zeplin design as shown below 👇 
### Zeplin Design
![image](https://user-images.githubusercontent.com/46839548/201492413-f06c2da7-9977-4a1d-bed6-22bdd57af972.png)

### Your implementation
![image](https://user-images.githubusercontent.com/46839548/201492394-0c25715e-5652-416b-a813-d3d122b253d1.png)